### PR TITLE
BROKERS: Delete and Select post

### PR DESCRIPTION
**Pull request by ADILATIC**
Closes: #27 #28 #29 #30 #31

### CODE REVIEW

**Comment by ADILATIC** [comment]
Closes: #27 #28 #29 #30 #31

File: InternTrack.Core.Api/Brokers/Storages/StorageBroker.Intern.cs [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+// Copyright (c) Coalition of the Good-Hearted Engineers
+// FREE TO USE FOR THE WORLD
+// -------------------------------------------------------
+
+using InternTrack.Core.Api.Models.Interns;
+using Microsoft.EntityFrameworkCore.ChangeTracking;
+using Microsoft.EntityFrameworkCore;
+using System.Linq;
+using System.Threading.Tasks;
+using System;
```
Comment (hassanhabib) [inline InternTrack.Core.Api/Brokers/Storages/StorageBroker.Intern.cs:11]: For when you open the 5 PRs make sure you configure Visual Studio to clean up and move `System` directives to the top.

File: InternTrack.Core.Api/Brokers/Storages/IStorageBroker.Intern.cs [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+using System.Linq;
+using System.Threading.Tasks;
+using InternTrack.Core.Api.Models.Interns;
+
+namespace InternTrack.Core.Api.Brokers.Storages
+{
+    public partial interface IStorageBroker
+    {
+        ValueTask<Intern> InsertInternAsync(Intern intern);
+
```
Comment (hassanhabib) [inline InternTrack.Core.Api/Brokers/Storages/IStorageBroker.Intern.cs:11]: For your next PRs remove spaces between one-liners

File: InternTrack.Core.Api/Models/Interns/Intern.cs [hunk 3]
```diff
@@ -0,0 +1,9 @@
+﻿using System;
+
+namespace InternTrack.Core.Api.Models.Interns
+{
+    public class Intern
+    {
+        public Guid Id { get; set; }
```
Comment (hassanhabib) [inline InternTrack.Core.Api/Models/Interns/Intern.cs:7]: Wait for @Manasa1191's PR to merge in first.

**Pull request closed without merging**